### PR TITLE
feat: Move all the common APIs into go-mod-bootstrap

### PR DESCRIPTION
**Pull request by jinlinGuan**
related: edgexfoundry/go-mod-bootstrap#478

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: pkg/service/service.go [hunk 1]
```diff
@@ -225,6 +227,7 @@ func (s *deviceService) LoadCustomConfig(customConfig interfaces.UpdatableConfig
 	}
 
 	s.controller.SetCustomConfigInfo(customConfig)
```
Comment (lenny-intel) [inline pkg/service/service.go:229]: Can this now be removed since it was for the common Common endpoint?

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: pkg/service/service.go (hunk 1 above)
Comment (cloudxxx8) [reply pkg/service/service.go:229]: custom config is different from common config
it's for the special config in different device services

Comment (lenny-intel) [reply pkg/service/service.go:229]: Right... ;-) THX!

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**